### PR TITLE
Apply PHP 8.0 syntax

### PR DESCRIPTION
**Pull request by samsonasik**
Signed-off-by: Abdul Malik Ikhsan <samsonasik@gmail.com>

|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Since composer.json require php 8.0, php 8.0 syntax can be applied (typed properties), also protected properties on final class can be private.

### CODE REVIEW

**Comment by samsonasik** [comment]
All green :tada:

File: src/BlackHole.php [hunk 1]
```diff
@@ -42,24 +42,20 @@ final class BlackHole implements
...
-    protected $capabilityMarker;
+    private ?stdClass $capabilityMarker = null;
 
     /**
      * options
      *
      * @var null|AdapterOptions
      */
-    protected $options;
+    private $options;
```
Comment (Ocramius) [inline src/BlackHole.php:58]: Worth using a native type?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: src/BlackHole.php [hunk 2]
```diff
@@ -42,24 +42,20 @@ final class BlackHole implements
...
-    protected $capabilities;
+    private ?Capabilities $capabilities = null;
 
     /**
      * Marker to change capabilities
-     *
-     * @var null|object
      */
-    protected $capabilityMarker;
+    private ?stdClass $capabilityMarker = null;
```
Comment (Ocramius) [inline src/BlackHole.php]: IMO `object` 🤔

File: src/BlackHole.php (hunk 1 above)
Comment (samsonasik) [reply src/BlackHole.php:58]: Ok, updated with native types https://github.com/laminas/laminas-cache-storage-adapter-blackhole/pull/56/commits/876192e83e181d60f19850c27a451fea224e46a3

File: src/BlackHole.php (hunk 2 above)
Comment (samsonasik) [reply src/BlackHole.php]: ok, updated with object https://github.com/laminas/laminas-cache-storage-adapter-blackhole/pull/56/commits/876192e83e181d60f19850c27a451fea224e46a3

File: src/BlackHole.php (hunk 1 above)
Comment (samsonasik) [reply src/BlackHole.php:58]: it seems cause error in pslam as filled with mixed in `__construct()`

```
E░░░░░░░░░░
Error: src/BlackHole.php:83:33: PossiblyInvalidMethodCall: Cannot call method on possible non-empty-array<array-key, mixed> variable $this->options (see https://psalm.dev/113)
Error: src/BlackHole.php:83:33: PossiblyUndefinedMethod: Method Traversable::setAdapter does not exist (see https://psalm.dev/108)
Error: src/BlackHole.php:95:16: InvalidReturnType: The declared return type 'Laminas\Cache\Storage\Adapter\AdapterOptions' for Laminas\Cache\Storage\Adapter\BlackHole::getOptions is incorrect, got 'Laminas\Cache\Storage\Adapter\AdapterOptions|Laminas\Cache\Storage\Adapter\AdapterOptions&Traversable|Traversable|non-empty-array<array-key, mixed>' (see https://psalm.dev/011)
Error: src/BlackHole.php:104:16: InvalidReturnStatement: The inferred type 'Laminas\Cache\Storage\Adapter\AdapterOptions|Laminas\Cache\Storage\Adapter\AdapterOptions&Traversable|Traversable|non-empty-array<array-key, mixed>' does not match the declared return type 'Laminas\Cache\Storage\Adapter\AdapterOptions' for Laminas\Cache\Storage\Adapter\BlackHole::getOptions (see https://psalm.dev/128)

```

Rolling back https://github.com/laminas/laminas-cache-storage-adapter-blackhole/pull/56/commits/9f8f1d4edcd6c06493f201b5790aa6f2352d535a

**Comment by samsonasik** [comment]
All green :tada:

File: src/BlackHole.php (hunk 1 above)
Comment (Ocramius) [reply src/BlackHole.php:58]: Fair enough: let's keep it for now :)

**Review by Ocramius — APPROVED** [review]
🚢 Thanks @samsonasik!

**Pull request merged**